packages/core/src/http-handler.ts

```diff
@@ -60,6 +62,22 @@ export class HttpHandler {
     return requestContext;
   }
 
+  protected _bindGetFromContext(context: Context): void {
+    context.bind('getFromContext').toDynamicValue(() => {
+      return async (key: string) => {
+        return await context.get(key);
+      };
+    });
+  }
+
+  protected _bindBindElement(context: Context): void {
+    context.bind('bindElement').toDynamicValue(() => {
+      return async (key: string, value: ValueOrPromise<BoundValue>) => {
```

Comment:
If you don't like `bind`, then I am proposing `createBinding` or `defineBinding`.